### PR TITLE
Add example script for importing customers

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

A farmers market in Australia needed to create 1600 customer records with a member tag. They exported a CSV from their mailing list to import these customers. It was quicker for us to write a little script then doing this task manually.

This is a custom script for one time use but also an example we can build on to create a better import tool. I've seen Slack conversations about similar situations in other countries that could benefit from such a script.

#### What should we test?

No test.

#### Release notes

Added example script for developers to bulk import customer records.

Changelog Category: Added

### CODE REVIEW

**Review by sauloperez — APPROVED** [review]
I'd be great if this was a rake task so that it gets listed when running `rake -T` and we don't miss it

**Comment by mkllnk** [comment]
> I'd be great if this was a rake task so that it gets listed when running rake -T and we don't miss it

Maybe that's personal preference? It's rare that I use `rake -T`. There are lots of tasks and I wouldn't notice them more than the script directory.

I did a bit of research of rake task vs scripts and the main learning was: Rake tasks are great if you manage dependencies between tasks. You can easily chain them together. But for a one-off script, there is no benefit while rake adds a bit more overhead.

Do you have more reasons for using a rake task here?

File: script/import-customers.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Read customer entries in CSV format from STDIN and create those records in
+# the database.
+#
+# This script was written for a once-off import. If we want to perform this
+# task more often, we can make it more flexible and eventually add a
+# feature to the user interface.
+require 'csv'
+
+# Willunga Farmers Market
+enterprise_id = 2545
```
Comment (Matt-Yorkley) [inline script/import-customers.rb:10]: Can't we allow this variable to be set dynamically when running the script?

**Comment by luisramos0** [comment]
Hi Maikel, I also think a rake task would be better. Do we really need to have scripts and tasks? Can we agree to just have tasks for all these things, that way we wouldnt need the script folder. For me it looks like rake tasks are very simple things with namespace and a description. Why would we not use it for all scripts?
This task in particular is related to our data model, in my opinion this must be unit tested so that if we change the model, we will know if the task needs to be updated.

**Comment by sauloperez** [comment]
I wouldn't add much @luisramos0 hasn't already. To me, it's about consistency and not having to deal with N ways of doing things. Rake tasks are a pretty powerful convention in the community already.

**Comment by mkllnk** [comment]
Okay, sounds like moving all application related Ruby scripts to rake tasks is the preference of the team. I don't want to invest any more work in this pull request though. I labelled it an "example script" because it's just an example how we can do this, a prototype. We can either merge it or close it. Any improvements can be done if we need a similar script again. We can also dig out the closed PR then.

**Comment by luisramos0** [comment]
Rethinking this PR. The main problem I see with this PR is actually that the script was executed in AUS. That makes the AUS instance unsupportable: how do we know the script is correct and will not introduce bugs in the app?

I think we need to be a bit more strict with these scripts and tasks, at least in the instances supported by the global team, it's a risk I don't think we should take.

I'd move in the direction of not allowing anyone to run scripts like this in an instance, except if there is a PR with a task that is unit tested. What do you think about this?

**Comment by mkllnk** [comment]
I understand your concerns about quality control. It's a risk that the Aus team was willing to take because of the urgency of the problem. It was the difference between having a big hub delivering food to people or not.

I agree though that this approach should be avoided. We need to find a way to get features like this through the global pipe more quickly. Other instances would appreciate a customer import as well.

And I do agree with unit testing. I should have done that. And I will do that before using this script again. Closing this for now.

**Comment by kirstenalarsen** [comment]
I don't think I understand the technicals of this, but going forward it would seem to me like - now that API is a 1st class citizen - we should consider adding tag to the api endpoint and using this https://app.swaggerhub.com/apis/luisramos0/the-open_food_network/0.1#/users/put_customers__customer_id__json

I have no idea if it works or how - I don't know if anyone is using PUT API calls and if we are ready for them to do so, but it definitely seems like the way forward for this. if we could get to / were at a point where @mkllnk doing things for aus customers outside of global time by updating and improving API endpoints were not 'frowned' upon then we would drop the scripts IN A FLASH!

**Pull request closed without merging**